CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
 project(ZMAP C)
-set(ZMAP_VERSION 2.1.1)
+set(ZMAP_VERSION DEVELOPMENT) # Change DEVELOPMENT to version number for release
```

Comment:
This was based on a discussion with @zakird. I'm fine switching this to an option instead if that's what's desired.